tests/disaster-recovery/sc_arbiter/test_shutdowns.py

```diff
@@ -0,0 +1,487 @@
+from ocs_ci.ocs.resources.pod import (
+    get_mon_pod_id,
+    get_ceph_tools_pod,
+    wait_for_pods_to_be_in_statuses,
+    get_debug_pods,
+    get_mon_pods,
+)
+from ocs_ci.helpers.sanity_helpers import Sanity
+from ocs_ci.ocs.exceptions import (
+    CommandFailed,
+    ResourceWrongStatusException,
+    TimeoutExpiredError,
+    CephHealthException,
+)
+from ocs_ci.utility.retry import retry
+from ocs_ci.ocs.cluster import get_mon_quorum_ranks, fetch_connection_scores_for_mon
+from ocs_ci.utility.utils import ceph_health_check
+
+log = logging.getLogger(__name__)
+
+
+class TestZoneShutdowns:
+    @pytest.fixture()
+    def init_sanity(self, request, nodes):
+        """
+        Initial Cluster sanity
+        """
+        self.sanity_helpers = Sanity()
+
+        def finalizer():
+            """
+            Make sure all the nodes are Running and
+            the ceph health is OK at the end of the test
+            """
+
+            # check if all the nodes are Running
+            log.info("Checking if all the nodes are READY")
+            nodes_not_ready = [
+                node.name for node in get_nodes() if node.status != "Ready"
+            ]
+            if len(nodes_not_ready) != 0:
+                try:
+                    nodes.start_nodes(nodes=nodes_not_ready)
+                except Exception:
+                    log.error(
+                        f"Something went wrong while starting the nodes {nodes_not_ready}!"
+                    )
+                    raise
+            else:
+                log.info("All nodes are READY")
+
+            # Validate all nodes are in READY state and up
+            retry(
+                (
+                    CommandFailed,
+                    TimeoutError,
+                    AssertionError,
+                    ResourceWrongStatusException,
+                ),
+                tries=30,
+                delay=15,
+            )(wait_for_nodes_status(timeout=1800))
+            log.info(
+                f"Following nodes {nodes_not_ready} were NOT READY, are now in READY state"
+            )
+
+            # check cluster health
+            try:
+                log.info("Making sure ceph health is OK")
+                self.sanity_helpers.health_check(tries=50)
+            except CephHealthException as e:
+                assert all(
+                    err in e.args[0]
+                    for err in ["HEALTH_WARN", "daemons have recently crashed"]
+                ), f"[CephHealthException]: {e.args[0]}"
+                get_ceph_tools_pod().exec_ceph_cmd(ceph_cmd="ceph crash archive-all")
+                log.info("Archived ceph crash!")
+                ceph_health_check(constants.OPENSHIFT_STORAGE_NAMESPACE, tries=5)
+
+        request.addfinalizer(finalizer)
+
+    @pytest.mark.parametrize(
+        argnames="iteration, immediate, delay",
+        argvalues=[
+            pytest.param(1, False, 3),
+            # pytest.param(3, True, 3),
```

Comment:
is this commented code required?

Reply:
yeah thats needed. I will uncomment that part